src/Components/ImagesTable/EmptyState.tsx

```diff
@@ -52,72 +55,89 @@ const EmptyBlueprintsImagesTable = () => (
 );
 
 const EmptyImagesTable = () => {
+  const { isBeta } = useGetEnvironment();
+  const experimentalFlag = useExperimentalFlag();
   return (
     <Bullseye>
       <EmptyState variant={EmptyStateVariant.lg} data-testid="empty-state">
-        <EmptyStateHeader
```

Comment:
Wouldn't we still want to render this for the original images table? I.e. `{ !experimentalFlag && ... }`?

Reply:
Didn't realize `isBeta()` is already checked in `useExperimentalFlag` hook :sweat_smile: Fixed! I'll go through the code and check the other places I've recently updated.

What do you mean by "render this" in original imagesTable? The header? Currently it should be split so it's the same empty state for ImagesTable as it was, so "Create an RPM-DNF image" and for blueprints it will show the "No images" as per Kelsea's mocks.

Reply:
:facepalm: I missed the ternary operator, I understand now. Ignore me. :sweat_smile: